### PR TITLE
Use matplotlib if mayavi is not available

### PR DESCRIPTION
**Pull request by alexfikl**
Produces something like [this](https://i.imgur.com/1bB0maE.png).

### CODE REVIEW

**Review by inducer — COMMENTED** [review]
Thanks! Could you rewrite history to get the .png out? Otherwise LGTM.

File: modepy/tools.py [hunk 1]
```diff
@@ -474,23 +474,43 @@ def estimate_lebesgue_constant(n, nodes, visualize=False):
     lebesgue_worst = np.sum(np.abs(eq_to_out), axis=1)
     lebesgue_constant = np.max(lebesgue_worst)
 
-    if visualize:
+    if visualize and dims == 2:
```
Comment (alexfikl) [inline modepy/tools.py:477]: Should this raise if visualize is `True` for other dimensions?

Comment (inducer) [reply modepy/tools.py:477]: Yeah, probably.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**